lib/data/models/wallet_contract_type.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:json_annotation/json_annotation.dart';
+
+@JsonEnum(fieldRename: FieldRename.pascal)
```

Comment:
Why pascal case? :)

Reply:
As I understand, this needs for compatible with browser calls when converting to json

Reply:
Okay